sw/device/lib/handler.h

```diff
@@ -16,4 +16,70 @@ typedef enum exc_id {
   kIdMax = 31
 } exc_id_t;
 
+// Ibex only supports vectored interrupts, but the handler will not appear in
+// the interrupt vector table. Instead, the interrupt vector table will include
+// `j <interrupt_handler_name>`. The only requirement on the symbol in the jump
+// is that it must be correctly aligned. As Ibex supports the C extension, this
+// just means aligned to the nearest 2 bytes.
+//
+// You only need to use this ABI for handlers that are the first function called
+// in an interrupt handler. Subsequent functions can just use the regular RISC-V
+// calling convention.
+#define INTERRUPT_HANDLER_ABI __attribute__((aligned(2), interrupt))
+
+/**
+ * Default exception handler. Can be overidden.
```

Comment:
I would replace "can be overriden" with something more specific (i.e., "this is a weak symbol which can be overriden by providing an additional definition at link time" or something), especially because the weak annotation appears in the translation unit.